src/radical/saga/utils/pty_shell_factory.py

```diff
@@ -231,7 +231,7 @@ def _initialize_pty (self, pty_shell, info, posix=None) :
             prompt     = info['prompt']
             logger     = info['logger']
             latency    = info['latency']
-            timeout    = info['ssh_timeout']
+            timeout    = float(info['ssh_timeout'])
 
```

Comment:
we then should also cast `latency`?  Any idea why that suddenly is a string?  Otherwise this looks good.